test/src/kbasesearchengine/test/parse/ObjectParserTest.java

```diff
@@ -0,0 +1,256 @@
+package kbasesearchengine.test.parse;
+
```

Comment:
Can the API be reduced - e.g. are there methods that can be made private to reduce the size of the API? Are there any unused variables in the method signatures that could be removed?

Reply:
same here